packages/fx-core/resource/package.nls.json

```diff
@@ -1098,6 +1098,8 @@
   "driver.botAadApp.log.startExecuteDriver": "Executing action %s",
   "driver.botAadApp.log.successExecuteDriver": "Action %s executed successfully",
   "driver.botAadApp.log.failExecuteDriver": "Failed to execute action %s. Error message: %s",
+  "driver.botAadApp.log.successCreateBotAad": "Created Azure Active Directory application with client id %s.",
+  "driver.botAadApp.log.useExistedBotAad": "Used existed Azure Active Directory application with client id %s.",
```

Comment:
Maybe "existing" rather than "existed"? I think "existing" is more suitable here.